warehouse/query-core/src/test/java/datawave/query/jexl/visitors/LiteralNodeVisitorTest.java

```diff
@@ -0,0 +1,55 @@
+package datawave.query.jexl.visitors;
+
+import com.google.common.collect.Multimap;
+import datawave.query.jexl.JexlASTHelper;
+import org.apache.commons.jexl2.parser.ASTJexlScript;
+import org.apache.commons.jexl2.parser.ParseException;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class LiteralNodeVisitorTest {
```

Comment:
Let's include some test cases that exercise functions (include, exclude at a minimum) and some marker nodes (value exceeded, term exceeded, delayed)